src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -11774,8 +11772,7 @@ static void Test2()
                 );
         }
 
-        // PROTOTYPE(NullableReferenceTypes): Calculate lamba conversion.
-        [Fact(Skip = "TODO")]
+        [Fact]
```

Comment:
Consider removing `Main` methods from this test and others like it.

Reply:
Unfortunately, removing the `Main` methods would require updating the locations of all the diagnostics.

---
In reply to: [192623033](https://github.com/dotnet/roslyn/pull/27283#discussion_r192623033) [](ancestors = 192623033)